src/style.css

```diff
@@ -2,0 +1,105 @@
+@font-face {
+  font-family: Avenir;
+  font-weight: 300;
+  font-style: normal;
+  src: url('./fonts/Avenir-Book.ttf') format('truetype');
+}
+
+@font-face {
+  font-family: Avenir;
+  font-weight: 600;
+  font-style: normal;
+  src: url('./fonts/Avenir-Heavy.ttf') format('truetype');
+}
+
+html {
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  font-family: Avenir, sans-serif;
+  font-size: 16px;
+}
+
+.search-form {
+  display: flex;
+  justify-content: center;
+  margin: 20px auto 0;
+}
+
+.search-form_big {
+  width: 100%;
+  height: 70px;
+}
+
+.search-form_small {
+  width: 100%;
+  height: 42px;
+}
+
+.search-form__input {
+  box-sizing: border-box;
+  width: 100%;
+  height: 100%;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  color: #3d4e61;
+  font-weight: 300;
+  font-family: inherit;
+  outline: none;
+}
+
+.search-form__input_big {
+  background-image: url(images/Search.svg);
+  background-repeat: no-repeat;
+  background-position: 25px 24px;
+  background-size: 19px 19px;
+  font-size: 16px;
+  padding-left: 62px;
+}
+
+.search-form__input_big:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.search-form__input_big:focus {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+  font-weight: 400;
+  background: url(images/Search.svg), linear-gradient(180deg, #fff, #f6f6f7);
```

Comment:
Ensure that the image path 'images/Search.svg' is correct and that the image exists in the specified directory.